packages/flutter/lib/src/material/app.dart

```diff
@@ -930,33 +955,44 @@ class _MaterialAppState extends State<MaterialApp> {
     final Color effectiveSelectionColor = theme.textSelectionTheme.selectionColor ?? theme.colorScheme.primary.withOpacity(0.40);
     final Color effectiveCursorColor = theme.textSelectionTheme.cursorColor ?? theme.colorScheme.primary;
 
+    Widget childWidget = widget.builder != null
+      ? Builder(
+          builder: (BuildContext context) {
+            // Why are we surrounding a builder with a builder?
```

Comment:
This comment needs to be updated since the AnimatedTheme is no longer "above" and because the AnimatedTheme is optional.

Do we even need the extra build if the AnimatedTheme isn't used?

Reply:
Woah... good catch I thought i did remove `AnimatedTheme`.